### PR TITLE
(bug-fix): properly parse .dk domains

### PR DESCRIPTION
**Pull request by eyJhb**
The examples in the repository, was some of the few domains in the entirety of Denmark, that didn't contain all the good information.
I've added folketinget.dk, which has a phone number example + names, etc. and politikken.dk, which has an email address as well.

Besides that, I've made a quick change in how the parser work, and made a special case for `.dk` domains.

If this does not live up to the code standard, then please do inform me :)

### CODE REVIEW

**Comment by likexian** [comment]
Hello @eyJhb 
Thanks for your works!

---
*If you think this repository is helpful, please share it with friends, thanks.*

**Comment by eyJhb** [comment]
I didn't expect this to be merged that quickly :D - I have however just discovered an oversight on my part.
I had assumed that `Attention` would always be an e-mail address, but that isn't always the case. Ie. see the below response.

```
$ whois multisupps.dk
# Hello XXXXXXXXXXXXX. Your session has been logged.
#
# Copyright (c) 2002 - 2024 by Punktum dk A/S
#
# Version: 5.2.0
#
# The data in the DK Whois database is provided by Punktum dk A/S
# for information purposes only, and to assist persons in obtaining
# information about or related to a domain name registration record.
# We do not guarantee its accuracy. We will reserve the right to remove
# access for entities abusing the data, without notice.
#
# Any use of this material to target advertising or similar activities
# are explicitly forbidden and will be prosecuted. Punktum dk A/S
# requests to be notified of any such activities or suspicions thereof.

Domain:               multisupps.dk
DNS:                  multisupps.dk
Registered:           2024-01-12
Expires:              2025-01-11
Registrar:            Domeneshop AS
Registration period:  1 year
VID:                  no
DNSSEC:               Signed delegation
Status:               Active

Registrant
Handle:               ***N/A***
Name:                 FANDROPS AS
Attention:            FANDROPS AS
Address:              Retirovegen 2
Postalcode:           6416
City:                 MOLDE
Country:              NO
Phone:                +47 46460880

Nameservers
Hostname:             ns1.hyp.net
Hostname:             ns2.hyp.net
Hostname:             ns3.hyp.net
```

What would even be the proper field for that?  
Also I should note, it's never possible to know if it's an organization or registrant, because punktum.dk always returns the information they have as registrant. :)

Sorry for this however, that was not my intention ! I'm very glad for the quick merge, and this awesome project!

**Comment by eyJhb** [comment]
Also, I'm unsure if the phone number should be parsed into phone and phoneext, e.g. 

```
  Registrar: &whoisparser.Contact{
    ID:           "81",
    Name:         "Gandi SAS",
    Organization: "",
    Street:       "",
    City:         "",
    Province:     "",
    PostalCode:   "",
    Country:      "",
    Phone:        "+33.170377661",
    PhoneExt:     "",
    Fax:          "",
    FaxExt:       "",
    Email:        "abuse@support.gandi.net",
    ReferralURL:  "http://www.gandi.net",
  },
```

**Comment by likexian** [comment]
> I didn't expect this to be merged that quickly :D - I have however just discovered an oversight on my part. I had assumed that `Attention` would always be an e-mail address, but that isn't always the case. Ie. see the below response.
> 
> ```
> $ whois multisupps.dk
> # Hello XXXXXXXXXXXXX. Your session has been logged.
> #
> # Copyright (c) 2002 - 2024 by Punktum dk A/S
> #
> # Version: 5.2.0
> #
> # The data in the DK Whois database is provided by Punktum dk A/S
> # for information purposes only, and to assist persons in obtaining
> # information about or related to a domain name registration record.
> # We do not guarantee its accuracy. We will reserve the right to remove
> # access for entities abusing the data, without notice.
> #
> # Any use of this material to target advertising or similar activities
> # are explicitly forbidden and will be prosecuted. Punktum dk A/S
> # requests to be notified of any such activities or suspicions thereof.
> 
> Domain:               multisupps.dk
> DNS:                  multisupps.dk
> Registered:           2024-01-12
> Expires:              2025-01-11
> Registrar:            Domeneshop AS
> Registration period:  1 year
> VID:                  no
> DNSSEC:               Signed delegation
> Status:               Active
> 
> Registrant
> Handle:               ***N/A***
> Name:                 FANDROPS AS
> Attention:            FANDROPS AS
> Address:              Retirovegen 2
> Postalcode:           6416
> City:                 MOLDE
> Country:              NO
> Phone:                +47 46460880
> 
> Nameservers
> Hostname:             ns1.hyp.net
> Hostname:             ns2.hyp.net
> Hostname:             ns3.hyp.net
> ```
> 
> What would even be the proper field for that? Also I should note, it's never possible to know if it's an organization or registrant, because punktum.dk always returns the information they have as registrant. :)
> 
> Sorry for this however, that was not my intention ! I'm very glad for the quick merge, and this awesome project!

so, what is `FANDROPS AS` means here?

**Comment by eyJhb** [comment]
> > I didn't expect this to be merged that quickly :D - I have however just discovered an oversight on my part. I had assumed that `Attention` would always be an e-mail address, but that isn't always the case. Ie. see the below response.
> > ```
> > $ whois multisupps.dk
> > # Hello XXXXXXXXXXXXX. Your session has been logged.
> > #
> > # Copyright (c) 2002 - 2024 by Punktum dk A/S
> > #
> > # Version: 5.2.0
> > #
> > # The data in the DK Whois database is provided by Punktum dk A/S
> > # for information purposes only, and to assist persons in obtaining
> > # information about or related to a domain name registration record.
> > # We do not guarantee its accuracy. We will reserve the right to remove
> > # access for entities abusing the data, without notice.
> > #
> > # Any use of this material to target advertising or similar activities
> > # are explicitly forbidden and will be prosecuted. Punktum dk A/S
> > # requests to be notified of any such activities or suspicions thereof.
> > 
> > Domain:               multisupps.dk
> > DNS:                  multisupps.dk
> > Registered:           2024-01-12
> > Expires:              2025-01-11
> > Registrar:            Domeneshop AS
> > Registration period:  1 year
> > VID:                  no
> > DNSSEC:               Signed delegation
> > Status:               Active
> > 
> > Registrant
> > Handle:               ***N/A***
> > Name:                 FANDROPS AS
> > Attention:            FANDROPS AS
> > Address:              Retirovegen 2
> > Postalcode:           6416
> > City:                 MOLDE
> > Country:              NO
> > Phone:                +47 46460880
> > 
> > Nameservers
> > Hostname:             ns1.hyp.net
> > Hostname:             ns2.hyp.net
> > Hostname:             ns3.hyp.net
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > What would even be the proper field for that? Also I should note, it's never possible to know if it's an organization or registrant, because punktum.dk always returns the information they have as registrant. :)
> > Sorry for this however, that was not my intention ! I'm very glad for the quick merge, and this awesome project!
> 
> so, what is `FANDROPS AS` means here?

Sorry for not replying to you, holidays etc. `FANDROP AS` is the company name in this case. But e.g. for a private individual, the name is used as well. And I think you can setup attention for such a domain as well.

```
Domain:               jacob.dk
DNS:                  jacob.dk
Registered:           1997-05-22
Expires:              2029-06-30
Registration period:  2 years
VID:                  no
DNSSEC:               Signed delegation
Status:               Active

Registrant
Handle:               ***N/A***
Name:                 Jacob Jensen
Address:              Streetname 10
Postalcode:           2100
City:                 København Ø
Country:              DK
Phone:                +4588888888

Nameservers
Hostname:             ns1.simply.com
Hostname:             ns2.simply.com
Hostname:             ns3.simply.com
Hostname:             ns4.simply.com
```

TL;DR: The attention is not really the email, but it's just.. a freetext field (maybe), but I'm not sure where it fits in, in the contact. If it even fits in. Do you have any suggestions for where to put attention?
